doc/source/whatsnew/v1.2.0.rst

```diff
@@ -444,6 +444,7 @@ Groupby/resample/rolling
 - Bug in :meth:`Rolling.count` returned ``np.nan`` with :class:`pandas.api.indexers.FixedForwardWindowIndexer` as window, ``min_periods=0`` and only missing values in window (:issue:`35579`)
 - Bug where :class:`pandas.core.window.Rolling` produces incorrect window sizes when using a ``PeriodIndex`` (:issue:`34225`)
 - Bug in :meth:`RollingGroupby.count` where a ``ValueError`` was raised when specifying the ``closed`` parameter (:issue:`35869`)
+- Reimplemented :meth:`FixedWindowIndexer.get_window_bounds` in a simpler and more intuitive way (:issue:`36040`).
```

Comment:
This entry should describe the bug in #36040

Reply:
Ah I see, done